CMake/FindR.cmake

```diff
@@ -37,7 +39,7 @@ set(R_LIBRARIES ${R_LIBRARY_BASE})
 mark_as_advanced(RSCRIPT_EXECUTABLE R_LIBRARIES R_INCLUDE_DIR R_COMMAND R_LIBRARY_BASE)
 
 
-set( _REQUIRED_R_VARIABLES R_INCLUDE_DIR R_COMMAND )
+set( _REQUIRED_R_VARIABLES R_INCLUDE_DIR R_COMMAND R_VERSION_STRING)
```

Comment:
R_VERSION_STRING should not be listing in the require variables.

Line 50 needs to be updated. See https://cmake.org/cmake/help/v3.12/module/FindPackageHandleStandardArgs.html.

We need to switch to the second signature. Something like:

```
find_package( R
 REQUIRED_VARS ${_REQUIRED_R_VARIABLES} 
 VERSION_VAR R_VERSION_STRING )
```